repaygithub/cactus
# CACTUS-674 :: BrandBar Tests

Ticket: https://repayonline.atlassian.net/browse/CACTUS-674

As I was testing, I noticed that the `BrandBar.UserMenu` didn't support Home/End keyboard functionality, so I added that in real quick, since it was pretty easy. For some reason, when I put that logic in a `switch` statement, it broke the arrow functionality. It appeared that the inputs were being applied twice. For instance, one `ArrowDown` would result in the focus moving down two indices. That's why I kept it in an if-else-if chain.

Anyway, I think these tests should be more robust now, but if you notice anything I missed, let me know.

### Testing
1. Open the "BrandBar > Basic Usage" story
2. Open the UserMenu and use the Home/End keys. They should move the focus to the beginning/end of the list, respectively.
